packages/api/package.json

```diff
@@ -15,7 +15,9 @@
     "start": "node src/api-service.js",
     "dev": "nodemon -r dotenv/config src/api-service.js",
     "test": "echo \"Error: run tests from root\" && exit 1",
-    "postinstall": "if [ \"${NODE_ENV:-development}\" == \"development\" ]; then npx webpack-cli; else echo no webpack build; fi;"
+    "postinstall": "if [ \"${NODE_ENV:-development}\" == \"development\" ]; then npx webpack-cli; else echo no webpack build; fi;",
+    "database:migration:create": "npx sequelize-cli migration:generate --name migration; echo \"\nPlease change generated file extension from .js to .cjs before running migrate script.\";",
+    "database:migration:migrate": "npx sequelize-cli db:migrate --url \"postgres://${POSTGRES_USER}:${POSTGRES_PW}@${POSTGRES_HOST}/${POSTGRES_DB}\""
```

Comment:
Can we add a script that calls `db:migrate:undo`? We're writing a `down` method in our migrations so we may as well add a script to use it 🙂

Reply:
Sure thing